### PR TITLE
Translating thai documentation

### PR DESCRIPTION
**Pull request by timelessnesses**
## What kind of change does this PR introduce?

A documentation update in thai.  
- Translated headers.
- Fixing the picture link.
- Translate some more words

## What is the current behavior?

- Some headers isn't translated
- At line 50 the picture URL is not found
- Some word can be translated

## What is the new behavior?

- Translated the header
- Fix the URL in line 50
- Translated any transalatable word

## Additional context

Add any other context or screenshots.

### CODE REVIEW

**Comment by jerriep** [comment]
@aaronamm I see on Twitter they asked for someone who can speak Thai to review this PR for them. Perhaps you can assist?

**Comment by aaronamm** [comment]
@jerriep Thanks for tagging me. To be honest, I am not good at translation but I think my friend @dmakeroam can help out for this.
This PR looks good to me but I have only some concern as following:
-  Should we keep some original English words  and not spell them with Thai letters?
- I am not fully understand the reason of removing [this block](https://github.com/supabase/supabase/pull/4890/files#diff-cfef2091d796f1c03d40bc4818c89be1696048d71a04f05fbb95cee4cbe7d575L69-L75).

@dmakeroam I am pretty sure you can help reviewing this PR. Thanks.

**Comment by dmakeroam** [comment]
Let me take a look

**Comment by awalias** [comment]
looks like I can't assign you as reviewer @dmakeroam but can just comment here and I'll approve if good 👍

File: i18n/README.th.md [hunk 1]
```diff
@@ -16,15 +16,15 @@
...
 เข้าไปที่ [supabase.io/docs](https://supabase.com/docs) สำหรับเอกสารข้อมูล (Documentation) ฉบับเต็ม
 
-## Community & Support
+## ชุมชนและการสนับสนุน
 
-- [Community Forum](https://github.com/supabase/supabase/discussions). สำหรับ: ความช่วยเหลือในการใช้งาน Supabase หรือการพูดคุยแลกเปลี่ยนข้อมูลเกี่ยวกับ Databases Best Practices
+- [ฟอรั่มชุมชน](https://github.com/supabase/supabase/discussions). สำหรับ: ความช่วยเหลือในการใช้งาน Supabase หรือการพูดคุยแลกเปลี่ยนข้อมูลเกี่ยวกับ Databases Best Practices
 - [GitHub Issues](https://github.com/supabase/supabase/issues). สำหรับ: Bugs และ Erros ที่พบในการใช้งาน Supabase
-- [Email Support](https://supabase.com/docs/support#business-support). สำหรับ: ปัญหาที่พบสำหรับ Database หรือ Infrastructure ของคุณ
+- [การสนับสนุนทางอีเมล](https://supabase.com/docs/support#business-support). สำหรับ: ปัญหาที่พบสำหรับ Database หรือ Infrastructure ของคุณ
```
Comment (vtno) [inline i18n/README.th.md:27]: What if we use this instead?

```suggestion
- [การช่วยเหลือทางอีเมล](https://supabase.com/docs/support#business-support). สำหรับ: ปัญหาที่พบสำหรับ Database หรือ Infrastructure ของคุณ
```

It's more along the line of "getting help here".

File: i18n/README.th.md [hunk 2]
```diff
@@ -47,15 +47,15 @@ Supabase สร้างขึ้นมาจากการรวมกัน
...
 
-![สถาปัตยกรรม](https://supabase.com/assets/images/supabase-architecture-9050a7317e9ec7efb7807f5194122e48.png)
+![สถาปัตยกรรม](https://supabase.com/docs/assets/images/supabase-architecture-9050a7317e9ec7efb7807f5194122e48.png)
 
 - [PostgreSQL](https://www.postgresql.org/) เป็นระบบฐานข้อมูลแบบ object-relational ที่ถูกพัฒนาอย่างต่อเนื่องมากว่า 30 ปี และเป็นที่ยอมรับในเรื่องของเสถียรภาพ ความมั่นคง และประสิทธิภาพ
-- [Realtime](https://github.com/supabase/realtime) เป็นเซิร์ฟเวอร์ที่พัฒนาขึ้นด้วย Elixir ที่ทำหน้าที่ตรวจจับเมื่อมีการเปลี่ยนแปลงของข้อมูล (inserts, updates, and deletes operation) ใน Postgres ด้วย websockets. Supabase คอยตรวจจับเมื่อ Postgres มีการทำซ้ำ (replication) ทำการแปลงข้อมูลเป็น JSON และกระจายข้อมูลผ่าน websockets
+- [Realtime](https://github.com/supabase/realtime) เป็นเซิร์ฟเวอร์ที่พัฒนาขึ้นด้วย Elixir ที่ทำหน้าที่ตรวจจับเมื่อมีการเปลี่ยนแปลงของข้อมูล (แทรก อัปเดต และลบ) ใน Postgres ด้วย websockets. Supabase คอยตรวจจับเมื่อ Postgres มีการทำซ้ำ (replication) ทำการแปลงข้อมูลเป็น JSON และกระจายข้อมูลผ่าน websockets
```
Comment (vtno) [inline i18n/README.th.md:53]: I think the original ones make more sense. It is quite normal to use English word for technical related topic. Translating it to Thai makes it more confusing.

```suggestion
- [Realtime](https://github.com/supabase/realtime) เป็นเซิร์ฟเวอร์ที่พัฒนาขึ้นด้วย Elixir ที่ทำหน้าที่ตรวจจับเมื่อมีการเปลี่ยนแปลงของข้อมูล (ทั้งการ insert, update และ delete) ใน Postgres ด้วย websockets. Supabase คอยตรวจจับเมื่อ Postgres มีการทำซ้ำ (replication) ทำการแปลงข้อมูลเป็น JSON และกระจายข้อมูลผ่าน websockets
```

File: i18n/README.th.md [hunk 3]
```diff
@@ -47,15 +47,15 @@ Supabase สร้างขึ้นมาจากการรวมกัน
...
 - [PostgREST](http://postgrest.org/) เป็น web server ที่ทำหน้าที่ในการแปลงฐานข้อมูล Postgres ให้กลายเป็น RESTful API โดยตรง
 - [Storage](https://github.com/supabase/storage-api) ทำหน้าที่ในการเป็น RESTful interface สำหรับการจัดการไฟล์ที่ถูกเก็บใน S3 และใช้ Postgres ในการจัดการ permissions
 - [postgres-meta](https://github.com/supabase/postgres-meta) เป็น RESTful API สำหรับจัดการ Postgres ของคุณ ทำให้คุณสามารถเรียกดูตาราง (Database tables), เพิ่ม Roles และเรียกดูข้อมูล (run queries) ได้
 - [GoTrue](https://github.com/netlify/gotrue) เป็น API ที่สร้างขึ้นจาก SWT สำหรับการจัดการผู้ใช้ และสร้าง SWT tokens
-- [Kong](https://github.com/Kong/kong) เป็น cloud-native API gateway
+- [Kong](https://github.com/Kong/kong) เป็น เกตเวย์ API ดั้งเดิมบนคลาวด์
```
Comment (vtno) [inline i18n/README.th.md:58]: Same comment as the above.
```suggestion
- [Kong](https://github.com/Kong/kong) เป็น cloud-native API gateway
```

**Review by vtno — COMMENTED** [review]
Small adjustments to make the wording more natural.

**Review by kiwicopple — APPROVED** [review]

**Comment by kiwicopple** [comment]
Thanks for the help everyone! I'll merge all the suggestions and push to prod. If there are any other changes feel free to PR :)

Also - reach out to swag@supabase.io if you want some swag for helping here!

File: i18n/README.th.md (hunk 1 above)
Comment (timelessnesses) [reply i18n/README.th.md]: late

**Pull request merged**